src/include/utime.h

```diff
@@ -122,15 +123,15 @@ class utime_t {
   }
   void encode(bufferlist &bl) const {
 #if defined(CEPH_LITTLE_ENDIAN)
-    bl.append((char *)(this), sizeof(__u32) + sizeof(__u32));
+    bl.append((char *)(this), sizeof(time_t) + sizeof(long));
 #else
     ::encode(tv.tv_sec, bl);
     ::encode(tv.tv_nsec, bl);
 #endif
   }
   void decode(bufferlist::iterator &p) {
 #if defined(CEPH_LITTLE_ENDIAN)
-    p.copy(sizeof(__u32) + sizeof(__u32), (char *)(this));
+    p.copy(sizeof(time_t) + sizeof(long), (char *)(this));
```

Comment:
changing the encoded format here without any kind of versioning support would make it impossible to upgrade from luminous to mimic without corrupting existing data